concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md

```diff
@@ -1,9 +1,74 @@
+![DevExtreme Download Manager](/images/Common/licensing-key@2x.png)
 
-    <!--JavaScript--> DevExpress.config({ 
-        // ... 
-        licenseKey?: string; 
-        // ... 
-    }); 
\ No newline at end of file
+When multiple developers work on the project, each of them has their own personal license key, which they must specify for every machine they work on.
+
+### Manual Configuration (Simplest Case)
+
+After you copy the key from Download Manager, create a file in which the license key will be stored.
+
+For example, the sources in your project are located under `src` folder. You can place the key in the `src/devextreme-license.ts` file. Create the file and paste the license key you copied from Download Manager:
+
+    <!-- tab: devextreme-license.ts -->export const licenseKey = 'DEVELOPER_LICENSE_KEY’;
+
+In order for each developer to use their own license key, the file should not be stored in the source code repository. Therefore, it is important to instruct Git to ignore the file that holds the key. To do this, you need to add the file path to your project's .gitignore file:
+
+    <!-- tab: .gitignore -->src/devextreme-key.ts
+
+This action also prevents your team from accidentally committing the key.
+
+DevExtreme license key must be entered within [GlobalConfig](/api-reference/50%20Common/Object%20Structures/GlobalConfig '/Documentation/ApiReference/Common/Object_Structures/GlobalConfig'). This should be done in the entry point of the application:
+
+    <!--JavaScript-->import config from 'devextreme/core/config'; 
+    import { licenseKey } from './devextreme-license'; 
+    
+    config({ licenseKey });   
+
+If your project does not use bundlers and includes sources via script tags, you can add a reference to a file holding the license key registration: 
+
+    <!--JavaScript--><script src="./dx.all.js" type="text/javascript"> </script> 
+    <script src="./devextreme-key.js" type="text/javascript"></script>
+
+    <!-- tab: devextreme-key.js --> DevExpress.config({ licenseKey: '' });
+
+### Automated License Key File Creation
+
+Instead of manually creating the `devextreme-license.ts` file to store the license key, you can use a script that will automatically generate the file. The following code is an example of what such a script might look like:
+
+    <!--JavaScript-->const fs = require('fs') 
+    const path = './src/devextreme-license.ts'; 
+    const key = process.env.DEVEXTREME_KEY ?? ''; 
+
+    if (key || !fs.existsSync(path)) { 
+        fs.writeFileSync(path, `export const licenseKey = '${key}';`); 
+    }
+
+As shown above, if the `src/devextreme-license.ts` file is absent, the script creates it and assigns the value of the `DEVEXTREME_KEY` environment variable as the key enable value. Utilizing the `DEVEXTREME_KEY` variable is beneficial in CI; however, if you wish to use it locally, ensure that you initialize it with the license key value before executing the script.
```

Comment:
if `DEVEXTREME_KEY` is set we use it to assign to the key, otherwise the key is an empty string. Should we highlight it or everybody can understand this piece of code?